manifests/slave.pp

```diff
@@ -170,6 +176,16 @@
     }
   }
 
+  if $slave_args {
+    if is_array($slave_args) {
+      $_combined_slave_args = hiera_array('jenkins::slave::slave_args', $slave_args)
```

Comment:
Whoa -- I feel this is pretty surprising behavior.  What is the motivation for doing this?